cmake/SetupLibraries.cmake

```diff
@@ -28,72 +28,6 @@ else()
     set(CARE_HAVE_BASIL "0" CACHE STRING "")
 endif()
 
-################################
-# UMPIRE
-################################
-if (UMPIRE_DIR)
-    include(cmake/libraries/Findumpire.cmake)
```

Comment:
If the user wants to have an external umpire (like if we are compiling ale3d), will things still work? Or does it only work with umpire as a submodule?

Reply:
It should work to use an external library in place of all the submodules. The paths for umpire_DIR, chai_DIR, etc..., need to actually point to the directory containing umpire-config.cmake, chaiConfig.cmake, etc....

Reply:
Do you think it would make sense to have CHAI_DIR etc for external users to be consistent with how other options and other packages work, and then copy those values to chai_DIR internally?

Reply:
I've made it so we can use CHAI_DIR or chai_DIR.